app/routes/photos/components/GalleryEditor.js

```diff
@@ -180,7 +181,7 @@ class GalleryEditor extends Component<Props, State> {
 
     return (
       <Content>
-        <NavigationTab title="Nytt album">
+        <NavigationTab title={`Redigerer: ${gallery.title}`}>
```

Comment:
`Editor` components are often used for both Adding and Editing, and this is no exception. This line causes a crash in `/photos/new`, as `gallery` is undefined. You could do something like `gallery ? gallery.title : 'Nytt album'`.

Reply:
I see, didn't know that component was used for adding :grin: